packages/itwin/reports-config-widget/src/widget/components/Reports.tsx

```diff
@@ -81,6 +80,9 @@ export const Reports = () => {
   const [isLoading, setIsLoading] = useState<boolean>(true);
   const [searchValue, setSearchValue] = useState<string>("");
   const [reports, setReports] = useState<Report[]>([]);
+  const bulkExtractor = useMemo(() =>
+    new BulkExtractor(apiConfig, reports.map((r) => r.id)), [apiConfig, reports]);
+  const jobStartEvent = new BeEvent<(reportId: string) => void>();
```

Comment:
This event should be memo'ized. Currently a new event will be created for each re-render.

Reply:
`jobStartEvent` now uses `useMemo`. Should there be any dependencies?

Reply:
No this should be enough.